demo/site/package.json

```diff
@@ -50,7 +50,9 @@
         "redraft": "^0.10.2",
         "styled-components": "^6.1.8",
         "swiper": "^11.1.15",
-        "ts-node": "^10.0.0"
+        "swr": "^2.2.5",
```

Comment:
Is that the best library to use? Do we need a library at all?

Reply:
The main thing why I'm asking is because it's called swr (stale-while-revalidate), and comes with a shared cache that supports swr. In our typical use cases I don't think we need (a) a shared cache or (b) stale-while-revalidate.

> Do you suggest an alternative?

Maybe a hook implemented in application code would be enough? But I don't have any recent real world expericence.